config/options.go

```diff
@@ -1188,6 +1188,14 @@ func (o *Options) GetCookieSecret() ([]byte, error) {
 		}
 		cookieSecret = string(bs)
 	}
+
+	if IsAll(o.Services) && cookieSecret == "" {
+		cookieSecret = randomSharedKey
```

Comment:
Maybe warn saying if the user reload without this being set; this will break all active sessions and link to our docs on how to set this. Because this will be a footgun / users will trigger this without realizing why it’s happening.